<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileRulesDialog</class>
 <widget class="QDialog" name="FileRulesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件审计规则设置</string>
  </property>
  <widget class="QGroupBox" name="groupBox_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>800</height>
    </size>
   </property>
   <property name="title">
    <string>文件审计规则</string>
   </property>
   <widget class="QCheckBox" name="wcheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>w</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rcheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>35</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000000</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>文件路径：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>133</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>结束时间：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePathEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>221</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="xcheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>35</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>关键字：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>96</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>生效时间：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>权限：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="browFileButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="etTimeEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>MM/dd/yy hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="stTimeEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2017</year>
      <month>3</month>
      <day>31</day>
     </date>
    </property>
    <property name="time">
     <time>
      <hour>0</hour>
      <minute>3</minute>
      <second>0</second>
     </time>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>7999</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>MM/dd/yy hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kwlineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>231</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>200</y>
      <width>333</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>80</height>
     </size>
    </property>
    <property name="title">
     <string>文件追踪</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>追踪结果：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="trackRslineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="traceButton">
       <property name="text">
        <string>追踪</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="acheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>35</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
